src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/movements/ExternalMovementsConfiguration.kt

```diff
@@ -17,6 +17,8 @@ class ExternalMovementsConfiguration(
   @Value("\${api.base.url.ext.movements}") val apiBaseUri: String,
   @Value("\${api.health-timeout:2s}") val healthTimeout: Duration,
   @Value("\${api.timeout:90s}") val timeout: Duration,
+  @Value("\${api.ext-movements-mapping-timeout:20s}") val mappingTimeout: Duration,
```

Comment:
For some of the large prisoner migrations a timeout of 10s is not enough, so trying 20s.

Once I can get the mapping upserts working I can hopefully have a peek at the AppDependencies in app insights for clues to improve the query performance.